zds/settings/abstract_base/zds.py

```diff
@@ -68,6 +68,8 @@
         "la communauté.",
         "googleAnalyticsID": zds_config.get("google_analytics_id"),
         "googleTagManagerID": zds_config.get("google_tag_manager_id"),
+        "matomoID": zds_config.get("matomo_id", 3),
```

Comment:
Pinaillage pour la lisibilité des paramètres à long terme : le terme standard et utilisé absolument partout par Matomo pour l’identifiant du site, c'est `site_id`. Il serait bien de l'utiliser aussi, ce serait plus clair pour quelqu'un qui verrait le code sans avoir vu cette PR.

Bien sûr ça impliquer d'adapter l'usage de ce paramètre.

```suggestion
        "matomoSiteID": zds_config.get("matomo_site_id", 3),
```